L/libunicorn/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libunicorn"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/oblivia-simplex/unicorn.git", "cb2c6c18e713806aeedbc4a71e18f541b77cd903")
```

Comment:
Just wondering what the reason is for using this fork instead of https://github.com/unicorn-engine/unicorn ?

Reply:
There's a race condition bug in the upstream version. I just submitted a patch to them. Once it's merged, I'll update this wrapper.